README.md

```diff
@@ -0,0 +1,22 @@
+# Aries Mobile Agent Android
+
+## Installation
+* Add Hyperledger Maven repo `maven { url "https://repo.sovrin.org/repository/maven-public" }` in build.gradle
+* Set minSdkVersion to 21 in build.gradle
+* Import Aries Mobile Agent Android AAR
```

Comment:
Ahh we do need to add the dependency library/AAR import in the App Module gradle too